### ISSUE TITLE
Compiling LAPKT repository (Ubuntu)

### ISSUE THREAD

**Issue by yotamitai** [LAPKT-dev/LAPKT-public#23]
Hi,
I'm trying to setup your planner on my Ubuntu PC and I'm having a hard time getting it to work.
(I'm not interested in using the docker image)
Cloning and building LAPKT as described up to section 3 works well, but from then on it is unclear as to what needs to be compiled/built in order to actually run the planner.
I would like to get to a point where i can use the siw executable as described in the docker documentation:
`./siw --domain domain.pddl --problem .problem.pddl`
I have tried going to the siw directory in /planners and using the scons command to compile it. The compilation succeeds but still there is no siw executable. when running build.py i receive the following error: 
fatal error: google/profiler.h: No such file or directory

I've tried installing the google pprof tool but that hasn't resolved the issue.

In short- I'd love it if you could provide me with a little more detail regarding the correct setup of your planner.

Thanks,
Yotam

**Comment by miquelramirez** [comment]
Hello @yotamitai,

the planners can be build via the scripts in the ```planners``` folder. For each of the planners we have, there are two variants: one that uses ```FF`` parser as a front end, and another that uses ```FastDownward``` as a parser. From the command you're mentioning above, I guess that you're after the ```FF``` variant.

If that is the case, you should get into the folder ```planners/siw-ff-parser``` and issue the command 

```
$ scons
``` 

to initiate the build process. You need to have compiled already the library either with optimizations or debug symbols as per Section 2.1 of the Readme.

Section 3 walks you through the examples that illustrate the use use of LAPKT as a library.
